R/fdbk_asdataframe.R

```diff
@@ -1156,6 +1156,58 @@ comparableRows <- function(DT,splitCol,splitVal,compareBy){
 ########################################################################################################################
 
 
+#' Bin a data.table column around user defined levels and replace it with the levels value.
+#' 
+#' @description Other way to perform a binning like in function fdbk_dt_binning but by defining levels around which to bin instead of the bins limits. 
```

Comment:
Could you please reflow the comments, such that line lengths are shorter and readable without scrolling?